### PR TITLE
Add SciDAVis v1.23

### PR DESCRIPTION
**Pull request by fjperini**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[closed issues]: https://github.com/Homebrew/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/scidavis.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  appcast 'https://sourceforge.net/projects/scidavis/rss.xml'
+  name 'scidavis'
+  homepage 'http://scidavis.sourceforge.net/'
+
+  depends_on macos: '>= :yosemite'
+
+  pkg "scidavis-#{version}.pkg"
+
+  uninstall pkgutil: 'SciDAVis'
+  # delete:  '/Applications/scidavis.app'
```
Comment (commitay) [inline Casks/scidavis.rb]: This line can be removed as the app is removed by `pkgutil`

File: Casks/scidavis.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  url "https://downloads.sourceforge.net/scidavis/scidavis-#{version}.pkg"
+  appcast 'https://sourceforge.net/projects/scidavis/rss.xml'
+  name 'scidavis'
+  homepage 'http://scidavis.sourceforge.net/'
+
+  depends_on macos: '>= :yosemite'
+
+  pkg "scidavis-#{version}.pkg"
+
+  uninstall pkgutil: 'SciDAVis'
```
Comment (vitorgalvao) [inline Casks/scidavis.rb]: Are you sure?

File: Casks/scidavis.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  # delete:  '/Applications/scidavis.app'
+
+  # zap trash: '~/Library/Saved Application State/net.sourceforge.scidavis.savedState'
```
Comment (commitay) [inline Casks/scidavis.rb]: This should be uncommented.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/scidavis.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/scidavis.rb]: Why the commented out lines? They should either be included or not, never commented out without a strong reason.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/scidavis.rb (hunk 2 above)
Comment (commitay) [reply Casks/scidavis.rb]: https://travis-ci.org/Homebrew/homebrew-cask/builds/403312643#L450

File: Casks/scidavis.rb (hunk 3 above)
Comment (fjperini) [reply Casks/scidavis.rb]: The formula works correctly. The commented lines can be eliminated.

Comment (fjperini) [reply Casks/scidavis.rb]: It must be uncommented? Ok, I make those modifications.

**Review by fjperini — COMMENTED** [review]
The changes were made! :+1:

**Pull request merged**